packages/example/package.json

```diff
@@ -19,7 +18,6 @@
     "tsx": "^4.16.5"
   },
   "devDependencies": {
-    "@bufbuild/buf": "^1.35.1",
```

Comment:
> This was the only place we had @bufbuild/buf as a dep, which didn't feel right. I moved it into the top-level package.json since it seemed like a dev dependency for the whole repo really. Let me know if you'd rather only include it in the packages it's used.

Yeah, not good style to have the dependency nested here, and use it at many other places.

We _do_ need it here, though. Otherwise, the example won't work as expected when following the instructions in the README.

Can you add the dependency to every package that uses it instead? We do that in protobuf-es as well.